sql/2019/04_Media/04_11.sql

```diff
@@ -0,0 +1,61 @@
+/*
+standard SQL
+04_11
+265 GB
+
+bytes per pixel photographic
+
+get the natural height/width, as well as the height/width on the pag
+from the custom query.  There's some fun JSON parsing, asn the nested JSON has encoded characters
+
+I then pull in the response size for each image.
+I calculate the # of pixels on the screen (imgh*imgw) and use that to calculate bytes per pixel
+to ensure photographic images, I only allow images >50KB 
+
+I then take the quantiles of this total for a chart.
+
+Looking at the extremes - there is some error - lots of images classigied as h=1 w=1 (when not defined on the page)
+and some are classifed as WAY too large as well.
+*/
+
+select approx_quantiles(byteperpixelonscreen, 1000)from(
+
+select DISTINCT(imgurl),hratio, wratio, pixelsOnScreen, totalPixels, respsize,
```

Comment:
Lots of unused values here that need to be trimmed.

Reply:
removed a few (commented out)  I may need them as I write....